### DISCUSSION TITLE
Get 100% in test AD block with adguardhome ( with blocklists) without app for browser

### DISCUSSION THREAD

**Discussion by ganganrobert** [AdguardTeam/AdGuardHome#6686] [Q&A]
I made some tests and get 95% in https://d3ward.github.io/toolz/adblock.html  with adguardhome without uBlock origin in browser

<details><summary>Details</summary>
<p>

![Screenshot 2024-01-27 at 17-47-11 Test Ad Block - Toolz](https://github.com/AdguardTeam/AdGuardHome/assets/76208862/9e60a234-1471-4a54-8164-18419066fa6b)

</p>
</details> 

My question, is it possible without browser app only with adguardhome with block lists( when I turn on uBlock origin in browser- I get 100%) get 100% in this test?

**Comment by iam-py-test** [comment]
That test is highly inaccurate, and use of it is discouraged by uBlock Origin's developer [1].
 Firstly, no test can replicate a real-world scenario. Additionally, some of the domains "tested" either are not adservers, or are not the specific domains used to serve the ads [2, 3]. For example, `widgets.pinterest.com` does not appear to serve ads. Furthermore, it is impossible for the test to accurately check if a domain is blocked, due to the nature of browsers [1].

Thank you.

[1] https://twitter[.]com/gorhill/status/1583581072197312512
[2] https://twitter[.]com/Yuki27183/status/1583821193110749186
[3] https://github.com/d3ward/toolz/issues/31

**Comment by ganganrobert** [comment]
Ok, thank you @iam-py-test but is it possible to block all ads on rambler.ru (with adguardhome), for example without plugins in browser?
<details><summary>Screenshot</summary>
<p>

![Screenshot 2024-01-28 at 21-46-04 Рамблер_новости почта и поиск — медийный портал новости России и мира электронная по](https://github.com/AdguardTeam/AdGuardHome/assets/76208862/b237f3ec-4146-4af2-9b2a-cc6a0ec7cdee)


</p>
</details>

**Reply by thiagodebastos** [reply]
Yes it is, give it a shot. I set it up on Linux with Docker.It's not as good as Ublock Origin since it is only dns blocking, but it does the job. It's especially good if you can tunnel your other devices through it.
<details><summary>Screenshot</summary>
![screenshot_29-Oct-2024_23-27-45](https://github.com/user-attachments/assets/bd38caad-4e57-4149-97a7-d560b29ece58)
</details>
